Yousei.Connectors/Http/HttpConnection.cs

```diff
@@ -23,6 +23,9 @@ public HttpConnection(HttpListener listener)
         private IObservable<HttpRequest> GetHttpRequests()
             => Observable.Create<HttpRequest>(async (observer, cancellationToken) =>
             {
+                if (listener is null)
+                    throw new InvalidOperationException($"HttpListener is not available.");
```

Comment:
In other files you use ArgumentNullException, do you use InvalidOperationException because it's more verbose on what's going on?

Reply:
It's not an argument here. The variable is still valid to have the value `null` but it's invalid to use it. (Only relevant when using the trigger)